File: ws/otavalo/u1/WS04Dependences/ChickenFarmSimulator/src/ec/edu/ec/chickenfarmsimulator/model/Egg.java
package ec.edu.ec.chickenfarmsimulator.model;

/**
 *
 * @author Arelys Otavalo
 */
public class Egg {
    private char size;

    @Override
    public String toString() {
        return "Egg{" + "size= " + size + '}';
    }
    
    public Egg(char size){
        this.size = size;
    }
          
    public char getSize() {
        return size;
    }

    public void setSize(char size) {
        this.size = size;
    }

    
}

File: ws/collaguazo/u1/ws06Json/egg.js
export class Egg {
  constructor(size) {
    this.size = size; // 'S', 'M', 'L'
  }

  getSize() {
    return this.size;
  }

  toString() {
    return `Egg(size=${this.size})`;
  }

  toJSON() {
    return {
      size: this.size
    };
  }
}
